bindings/wallet-cordova/www/wallet.js

```diff
@@ -307,18 +306,17 @@ var plugin = {
      * @param {Uint8Array} votePlanId a byte array of 32 elements that identifies the voteplan
      * @param {number} index the index of the proposal in the voteplan
      * @param {number} numChoices the number of choices of the proposal, used to validate the choice
-     * @param {Uint8Array} encryptionVoteKey a byte array of 65 elements, this
-     * is the single key used to encrypt a vote, generated from the public keys
+     * @param {Uint8Array} encryptionVoteKey bech32 string representing the
```

Comment:
Does the documented type need to be changed as well?

Reply:
yes